lib/Twig/Environment.php

```diff
@@ -1280,28 +1288,10 @@ protected function initExtension(Twig_ExtensionInterface $extension)
 
     protected function writeCacheFile($file, $content)
```

Comment:
hmm, we could call it only when overwritten though, and trigger a deprecation warning in this case (otherwise people overwriting it won't get the deprecation warning)